src/cascadia/TerminalControl/ControlCore.h

```diff
@@ -68,6 +68,7 @@ namespace winrt::Microsoft::Terminal::Control::implementation
     {
         til::property<Windows::Foundation::Collections::IVector<winrt::hstring>> History;
         til::property<winrt::hstring> CurrentCommandline;
+        til::property<Windows::Foundation::Collections::IVector<winrt::hstring>> QuickFixes;

```

Comment:
this needs to get initialized. It's the source of the suggestions crash I think - AppActionHandlers doesn't guard against it being null